### ISSUE TITLE
Please fork repo https://github.com/davidbpirie/autopkg-recipes

### ISSUE THREAD

**Issue by davidbpirie** [autopkg/autopkg#735]
Can you please fork my recipe repo at https://github.com/davidbpirie/autopkg-recipes. Ideally the name would be davidbpirie-recipes.

**Comment by homebysix** [comment]
Hi @davidbpirie — a couple questions before I create this repo:

- Your TeamViewerHost.download recipe seems identical to [the one already offered in hjuutilainen-recipes](https://github.com/autopkg/hjuutilainen-recipes/blob/master/TeamViewer/TeamViewer.download.recipe). Also, there appear to be existing options for pkg and munki recipes as well. Duplicate recipes are discouraged in the autopkg org, so if your recipes do something different than the existing public recipes, I'd recommend adding some information to your recipes' descriptions detailing the differences (see [this example](https://github.com/autopkg/MLBZ521-recipes/blob/49c542915b1777ae3c2ffbf8a179cefb34d19299/NoMAD/NoMADLoginAD.jss.recipe#L5-L8)).
- Similarly, there are already Rectangle.download recipes in [dataJAR-recipes](https://github.com/autopkg/dataJAR-recipes/tree/master/Rectangle) and [peetinc-recipes](https://github.com/autopkg/peetinc-recipes/tree/master/Rectangle).
- Thanks for including a pre-commit config! Although benign, I think this line isn't necessary in your version: `exclude: ^Docker\/|^Quicksilver\/Quicksilver\.pkg\.recipe$`

**Comment by homebysix** [comment]
Hi @davidbpirie - Just checking in on the above. Possible to consolidate those duplicates?

**Comment by davidbpirie** [comment]
Hi @homebysix. Apologies for the delay - I completely missed your replies.
I have removed the duplicates completely (TeamViewerHost and Rectangle). This is just me finally getting around to sharing recipes I wrote myself over the years and never made public, so in that time others have provided solutions, so I will just start using theirs instead.
Thanks also for the tip on the pre-commit config - this was my first use of pre-commit, and obviously made a copy/paste error there. I have removed the offending line.
Let me know if you would like me to make any more changes before you fork.
